.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,3 +24,4 @@ A clear and concise description of what you expected to happen and what actually
 ### Logs
 
 Paste the activity from your command line. Redact if needed.
+Tip: Set `GH_DEBUG=true` (verbose logs) or `GH_DEBUG=api` (verbose logs with HTTP traffic details)
```

Comment:
```suggestion

> [!NOTE]
> Set `GH_DEBUG=true` for verbose logs or `GH_DEBUG=api` for verbose logs with HTTP traffic details.
```

Reply:
I agree with your change, suggest just committing and merging this. Thank you for your contribution @TWiStErRob, I've been thinking we need to overhaul our bug report template for a while but this is an obvious win.

Reply:
@andyfeller merged to move on. I think it might be unnecessary to format as a callout, because it's going to be shown and consumed in markdown by the issue author. I might suggest that an HTML comment is enough here, the key is to make it show to the author, but not to others.

i.e.

```markdown
<-- Tip: ... -->
```

(Can't do a multiline proposal on my phone...)

Reply:
Will let @andyfeller make a decision about this and I'll just accept whatever.

Reply:
> @andyfeller merged to move on. I think it might be unnecessary to format as a callout, because it's going to be shown and consumed in markdown by the issue author. I might suggest that an HTML comment is enough here, the key is to make it show to the author, but not to readers of the issue.
> 
> Like this: https://github.com/cli/cli/pull/9167/files#r1642496276

Fair point 👍  Let's go with the HTML comment